gnustep-base/libobjc2/libobjc2-2.1.ebuild

```diff
@@ -1,42 +0,0 @@
-# Copyright 1999-2023 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=7
-inherit cmake
-
-DESCRIPTION="GNUstep Objective-C runtime"
-HOMEPAGE="https://gnustep.github.io"
-SRC_URI="https://github.com/gnustep/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
-	https://github.com/Tessil/robin-map/archive/757de82.tar.gz"
-
-LICENSE="MIT"
-SLOT="0"
-KEYWORDS="~amd64 ~x86"
-IUSE="boehm-gc test"
-RESTRICT="!test? ( test )"
-
-RDEPEND="boehm-gc? ( dev-libs/boehm-gc )"
-BDEPEND="${RDEPEND}
-	sys-devel/clang"
-
-PATCHES=(
-	"${FILESDIR}"/${P}-eh_trampoline.patch
-	"${FILESDIR}"/${P}-pthread_link.patch
-)
```

Comment:
These patches are not used in 2.2 so can be removed too

Reply:
thx